Source/player.cpp

```diff
@@ -1396,23 +1396,23 @@ void PM_ChangeOffset(int pnum)
 	}
 	auto &player = plr[pnum];
 
-	int px = player.position.offset2.x / 256;
-	int py = player.position.offset2.y / 256;
+	int px = player.position.offset2.deltaX / 256;
+	int py = player.position.offset2.deltaY / 256;
 
 	player.position.offset2 += player.position.velocity;
 
 	if (currlevel == 0 && sgGameInitInfo.bRunInTown != 0) {
 		player.position.offset2 += player.position.velocity;
 	}
 
-	player.position.offset = { player.position.offset2.x >> 8, player.position.offset2.y >> 8 };
+	player.position.offset = { player.position.offset2.deltaX >> 8, player.position.offset2.deltaY >> 8 };
 
-	px -= player.position.offset2.x >> 8;
-	py -= player.position.offset2.y >> 8;
+	px -= player.position.offset2.deltaX >> 8;
+	py -= player.position.offset2.deltaY >> 8;

```

Comment:
wonder if we should add `>>` as an operator or use `/` here instead

Reply:
This looks like a compiler optimised division. `/` gets my vote.

Reply:
Outside the scope of the PR.